public/faq.md

```diff
@@ -1 +1,100 @@
+
+- __Can I participate in KWoC as both a mentor and a student?__
+
+  We do encourage participation into project development from everyone; but if you are a mentor we won't be publishing your stats nor will you be going through evaluations.
+
+- __What can I do to help spread the word about KWoC?__
+
+  Nothing makes us happier than people spreading words about open-source; to begin, you can like our [facebook page](https://www.facebook.com/kossiitkgp) and increase awareness about KWoC in your college; nothing works better than word of mouth.
+
+- __What if I have a question not answered in this FAQ?__
+  
+  Please make sure to read the website,manual. If after reading through it you still don't have an answer please consider reaching out to us on admin@kossiitkgp.org.
+
+
+
+# STUDENT
+- __How should I get selected in KWOC?__
+
+  There is absolutely no selection criteria for students. You are selected in the program as soon as you register. So you don't have to ask your mentors "How to get selected for your project?", but instead go ahead with **"How can I contribute to your project?"**.
+
+- __My mentor is not replying, What should I do?__
+
+  Please be patient with your mentor(s). If you do not get a response on the communication channel, contact the mentor on their email id. If no response for 3 days, inform us.
+
+- __What is the last date of student registration?__
+
+  Students registration deadline is 6th December. The contribution chart goes live at the same time, so be quick. 
+  > early bird catches the worm :) .
+
+- __Can I show the contributions that I have made in KWoC for participating in any competitions?__
+  
+  All the contributions that you will make in KWoC will be displayed on your GitHub timeline. So, that is a big plus if you are participating in any competitions and your GitHub profile is on display.
+
+- __Can I apply if I am not a student of IIT Kharagpur?__
+
+  Sure ! Our program is open to __all__ students irrespective of college ! Is there any eligibility criteria and pre-requisites? There is no strict eligibility criteria as such. Read [this](https://www.quora.com/What-is-the-prerequisite-for-participating-in-Kharagpur-Winter-of-Code-2016) Quora answer by Pranit Bauva.
+
+- __Can you provide me any advice so that I can start preparing for it beforehand?__
+  
+  Play around git and GitHub. Create an account on GitHub, set up git on your machine and start experimenting with it. You can also begin by following Codecademy's git interactive tutorial.
+
+- __What are my chances of receiving a position in KWoC?__
+  Pretty fair ! You need to be working for the whole month regardless of your expertise level.
```

Comment:
What does this even mean? "position" as in rank? or certificate?

Reply:
oh,i didnot check properly.it is in faq last year.removing this irrelevant one